### ISSUE TITLE
Load server data?

### ISSUE THREAD

**Issue by lauriinc** [bigeyex/brackets-sftp-upload#50]
I'm using basically the same server data on all my projects but everytime I make a new project or rename a folder, I have to re-input all my server data again and again and again...

Is there any way to store / load server data? would be a great feature in the server setup window!

Loving this extension otherwise!

**Comment by claudericke** [comment]
I agree. Please add server profiles. This would be a great feature! Love your extension, keep up the good work.
